tests/test_session.py

```diff
@@ -1,43 +1,35 @@
 import pytest
-import respx
-import httpx
-from smartconnect import AsyncSmartClient, SMARTClientException
+import requests, requests_mock
+from smartconnect import SmartClient, SMARTClientException
 
-@pytest.mark.asyncio
-@respx.mock
-async def test_async_client_login():
+def test_async_client_login(requests_mock):
```

Comment:
Looks like this test was accidentally named as "async_client..", but it's for the sync client.